roles/repository/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-repository_keys_file: []
-repository_keys_file_dir: "/opt/configuration/environments/generic/repokeys"
+repository_key_files: []
+repository_key_files_dir: "/opt/configuration/environments/generic/repokeys"
```

Comment:
I would prefer to have only one parameter. Either _files with full path or only _dir and then all files in _dir are treated as keys and imported.

Reply:
with_items -> with_fileglob